src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,29 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
     <title>Login successfully</title>
+    <style>
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+        }
+
+        code {
+            font-family: 'Courier New', Courier, monospace;
```

Comment:
I would preferring `Consolas` as it is used by our official doc: https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=dnf

```suggestion
            font-family: Consolas, 'Liberation Mono', Menlo, Courier, monospace;
```